libs/contract/truffle-config.js

```diff
@@ -11,7 +11,10 @@ const { AccountId, getSigner } = require('@thecointech/signers');
 
 const numBuiltIn = AccountId.BrokerCAD + 1;
 const testAccounts = [];
-loadAccounts(numBuiltIn).then(v => testAccounts.push(...v)).catch(console.error);
+if (process.env.CONFIG_NAME !== 'devlive') {
+  // devlive accounts are hosted on our local blockchain, so already available
+  loadAccounts(numBuiltIn).then(v => testAccounts.push(...v)).catch(console.error);
+}
```

Comment:
This line would throw an error because the signers loaded are not wallets (remote signers from blockchain, and this code is loading them back onto the blockchain.  Very circular, and unnecessary)